README.md

```diff
@@ -524,8 +532,14 @@ All `actual_steps` with `name` "retrieval" contain:
 - `retrieval_answer_precision_reason`: (optional) LLM reasoning in evaluating `retrieval_answer_precision`
 - `retrieval_answer_precision_error`: (optional) error message if `retrieval_answer_precision` evaluation fails
 - `retrieval_answer_precision_cost`: cost of evaluating `retrieval_answer_precision`, in US dollars
-- `retrieval_answer_f1`: (optional) F1 score of the retrieved context with respect to the reference answer, if `retrieval_answer_recall_error` and `retrieval_answer_precision` succeed
+- `retrieval_answer_f1`: (optional) F1 score of the retrieved context with respect to the reference answer, if `retrieval_answer_recall` and `retrieval_answer_precision` succeed
 - `retrieval_answer_f1_cost`: The sum of `retrieval_answer_recall_cost` and `retrieval_answer_precision_cost`
+- `retrieval_context_recall`: (optional) recall of the retrieved context with respect to the reference answer, if evaluation succeeds
+- `retrieval_context_recall_error`: (optional) error message if `retrieval_context_recall` evaluation fails
```

Comment:
So, there are no "*_reason" fields, because currently ragas doesn't return them, right?

Reply:
Yes, unfortunately.

Should I check the other context evaluation LangEvals module?

Reply:
We could merge this as a working version and improve it later.